wrapanapi/systems/ec2.py

```diff
@@ -894,7 +894,7 @@ def find_templates(self, name=None, id=None, executable_by_me=True, owned_by_me=
 
     def get_template(self, name_or_id):
         return self._get_resource(name=name_or_id, resource=EC2Image,
-                                  find_method=self.find_templates)
+                                  find_method=self.find_templates, public=True)
```

Comment:
I'm not convinced this is the right approach here, defaulting this call to public=True.

Things to consider:
1. Timing impact of including the extension of all public images
2. Whether this should just pass `owned_by_me=False`, or even changing that default  in `find_templates` definition.
3. Allowing for passing of kwargs through this method, so that the caller can specify what scope should be used to find the template in.

Reply:
So I changed it to try to get private image first.
When it fails it tries to also get public images.